apiserver/apiserver.go

```diff
@@ -44,8 +44,9 @@ func NewAPIServer(o *Options) (*APIServer, error) {
 // ListenAndServe wraps http.Server and provides context-based cancelation.
 func (api *APIServer) ListenAndServe(ctx context.Context) error {
 	svr := http.Server{
-		Addr:    api.Options.Addr,
-		Handler: api.NewRouter(ctx),
+		Addr:              api.Options.Addr,
+		Handler:           api.NewRouter(),
+		ReadHeaderTimeout: 0,
```

Comment:
im assuming this is what it was doing before (deferring to ReadTimeout)

Reply:
(prompted by gosec saying not explicitly setting it is bad practice)

Reply:
0 is ignored